src/Model/Item.php

```diff
@@ -3027,9 +2994,43 @@ public static function performActivity($item_id, $verb)
 			}
 		}
 
```

Comment:
Can you add some `$activity = ''` in front of the case structure? My editor warns me because of an undefined variable. And although technically it can't happen that the variable isn't set, it wouldn't hurt as well.

Reply:
My IDE doesn't bark because of the `return` statement in the `switch`. I can still accommodate your ancient system.

Reply:
My "ancient" system is "Visual Studio Code" with "PHP Intelephense" in the newest version 😁